wordpress-mobile/WordPress-iOS
# Improve plugin list reloading

This adds a more selective model change notification to avoid reloading the full table every time the PluginStore state changes. This should fix the resizing icon problems we were seeing, as only the relevant row will reload when it's icon URL is retrieved.

I've tried to keep it simple for a first version, since the UI for this is changing soon. For future reference, we might want to look into [Dwifft](https://github.com/jflinter/Dwifft) or [IGListKit](https://github.com/Instagram/IGListKit)

To test:

- Load the plugin list for a site without cache
- Ensure all the icons look right
